### ISSUE TITLE
Server timezone does not match local timezone

### ISSUE THREAD

**Issue by jashephe** [SaturdayNeighborhoodHealthClinic/osler#247]
If you work on notes for clinic in the afternoon/evening (specifically after 6pm, given that we're currently in CST), SNHC says "Good morning, it looks like you're the first one here", etc. and has the date for Sunday. This is consistent with the server time being in UTC, not CST (6:01pm on Saturday CST is 12:01am on Sunday UTC).

**Comment by justinrporter** [comment]
Good catch. I thought we had this fixed. It's a bit more complicated than the server time vs the local time, because django is supposed to be time-zone aware. Typically, this type of problem crops up when python's `datetime.datetime.now()` is used instead of `django.utils.timezone.now()`, which is aware of the server's "local" timezone.
